deployment/terraform/versions.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 0.12.13"
```

Comment:
I think it is fine to use 0.12 here. We just need to be careful to keep this isolated from core, because the Terraform version used to manipulate state gets persisted in the state (and Terraform using prevents earlier versions from interacting with the state after that happens).

Reply:
I pinned this to 0.12.13 since I've been using that version to manipulate state. What you're saying would mean that we **can't** go back and pin this to 0.12, right?